src/lib/Components/Inbox/Conversations/MarkReadMessage.ts

```diff
@@ -1,9 +1,13 @@
 import { commitMutation, Environment, graphql, MutationConfig, RecordSourceSelectorProxy } from "react-relay"
-import { RelayProps } from "../../../Containers/Conversation"
+
+interface Conversation {
```

Comment:
The only thing this mutation needs is `id` (to mark the message as read), and also `__id`, to update the Relay store unread indicator.